client/js/components/addon/AddonWrapper.vue

```diff
@@ -0,0 +1,77 @@
+<license>
+  (c) 2010-present DEMOS E-Partizipation GmbH.
+
+  This file is part of the package demosplan,
+  for more information see the license file.
+
+  All rights reserved
+</license>
+
+<template>
+  <div>
+    <component
+      :is="component"
+      v-bind="getCurrentProperties()"
+    />
+  </div>
+</template>
+
+<script>
+import { checkResponse, dpRpc } from '@demos-europe/demosplan-utils'
+export default {
+  name: 'AddonWrapper',
+
+  props: {
+    hookName: {
+      type: String,
+      required: true,
+      default: ''
+    },
+
+    addonProps: {
+      type: Object,
+      required: false,
+      default: () => {}
+    }
+  },
+
+  data () {
+    return {
+      component: ''
+    }
+  },
+
+  methods: {
+    getCurrentProperties() {
```

Comment:
Why is a method needed here? Wouldn't it suffice to pass `addonProps` in the template?

Reply:
You are right, we can simply bind the  `addonProps` directly without needing a seperate function. Done in https://github.com/demos-europe/demosplan-core/pull/736/commits/b549b653dbeae08a97e5e3ebe385b097fe374b12